packages/cashier/src/Stores/Cashier/cashier-store.js

```diff
@@ -1018,10 +1018,10 @@ export default class CashierStore extends BaseStore {
         payment_agent_list.paymentagent_list.list.forEach(payment_agent => {
             this.config.payment_agent.list.push({
                 email: payment_agent.email,
-                phone: payment_agent.telephone,
+                phones: payment_agent.phone_numbers,
                 name: payment_agent.name,
                 supported_banks: payment_agent.supported_banks,
```

Comment:
`supported_banks` is deprecated and replaced by `supported_payment_agents`. It's not mentioned in the card's description, but please switch to the new field (again in a backward compatible way); if we don't do it now, it will be forgotten and client's will complain about misinformation.